test/integration/roles/setup_postgresql_db/tasks/main.yml

```diff
@@ -8,24 +8,10 @@
       paths: '../vars'
 
 # Make sure we start fresh
-- name: remove rpm dependencies for postgresql test
-  package: name={{ postgresql_package_item }} state=absent
-  with_items: "{{postgresql_packages}}"
-  loop_control:
-    loop_var: postgresql_package_item
-  when: ansible_os_family == "RedHat"
-
-- name: remove dpkg dependencies for postgresql test
-  apt: name={{ postgresql_package_item }} state=absent
-  with_items: "{{postgresql_packages}}"
-  loop_control:
-    loop_var: postgresql_package_item
-  when: ansible_pkg_mgr  ==  'apt'
-
 - name: remove old db (red hat)
```

Comment:
I know the motivation, but those tasks don't help there. Removing/Reinstalling the package does not really provide a clean start since at least on Ubuntu/Suse this won't remove the databases. Removing the databases/config/... is far less trivial.

Even with these tasks in, the tests are not idempotent (i.e. fail on the second run), which would be my minimal requirement for any test outside of docker.